$NetBSD$

Check if VLC was actually found (enough to build),
instead of just the library.

--- gemrb/plugins/VLCPlayer/CMakeLists.txt.orig	2016-02-29 18:00:08.000000000 +0000
+++ gemrb/plugins/VLCPlayer/CMakeLists.txt
@@ -1,5 +1,5 @@
-IF (LIBVLC_LIBRARY)
+IF (LIBVLC_FOUND)
 	INCLUDE_DIRECTORIES( ${LIBVLC_INCLUDE_DIR} )
 	ADD_GEMRB_PLUGIN ( VLCPlayer VLCPlayer.cpp VideoContext.cpp )
 	TARGET_LINK_LIBRARIES( VLCPlayer ${LIBVLC_LIBRARY} )
-ENDIF (LIBVLC_LIBRARY)
+ENDIF (LIBVLC_FOUND)
